NREL/ssc
# Add pvwattsv8 warning for bifacial with fixed roof mount

Fix https://github.com/NREL/SAM/issues/1276

To test:
1. Start SAM and create a PVWatts / No Financial case.
2. On the System Design page, under Advanced Inputs check **Module is bifacial**.
3. Run a simulation.

SAM should display a warning message about fixed roof mount not being appropriate for bifacial modules.
